rules.neon

```diff
@@ -22,6 +22,7 @@ rules:
 	- PHPStan\Rules\DisallowedConstructs\DisallowedImplicitArrayCreationRule
 	- PHPStan\Rules\DisallowedConstructs\DisallowedShortTernaryRule
 	- PHPStan\Rules\ForeachLoop\OverwriteVariablesWithForeachRule
+	- PHPStan\Rules\ForLoop\OverwriteVariablesWithForLoopInitRule
```

Comment:
Adding rule to `rules` automatically applies the rule tag, this needs to be added as a tag-less service in the `services` section.